sr_gui_change_controllers/src/sr_gui_change_controllers/change_controllers.py

```diff
@@ -276,13 +287,14 @@ def change_force_ctrl_type(self, chng_type_msg):
         it will deactivate the Effort, Position, Mixed and Velocity buttons for 3 secs to allow hardware controllers to be updated
         """
         success = True
-        change_control_type = rospy.ServiceProxy('realtime_loop/change_control_type', ChangeControlType)
-        try:
-            resp1 = change_control_type(chng_type_msg)
-            if resp1.result.control_type != chng_type_msg.control_type:
+        for hand_id in self.hand_ids:
+            change_control_type = rospy.ServiceProxy('realtime_loop/' + hand_id + '/change_control_type', ChangeControlType)
```

Comment:
```sub = rospy.Subscriber("/a//b", Float64, cb)
/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py:515: UserWarning: '/a//b' is not a legal ROS graph resource name. This may cause problems with other ROS tools